src/ui/components/Menu.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import React from "react";
+import {createElement} from "react";
+import {useSelector} from "react-redux";
+import {useMediaQuery} from "@material-ui/core";
+import {MenuItemLink, getResources} from "react-admin";
+import {withRouter} from "react-router-dom";
+import TrendingUpIcon from "@material-ui/icons/TrendingUp";
+import DefaultIcon from "@material-ui/icons/ViewList";
+
+type menuProps = {|onMenuClick: Function, logout: Function|};
+
+const Menu = ({onMenuClick, logout}: menuProps) => {
+  const isXSmall = useMediaQuery((theme) => theme.breakpoints.down("xs"));
+  const open = useSelector((state) => state.admin.ui.sidebarOpen);
```

Comment:
What would it take for us to get this type safe? Seems like there's typedefs available: https://github.com/flow-typed/flow-typed/tree/master/definitions/npm/react-redux_v7.x.x

Not for this PR, just something that would be good to add in soon

Reply:
I added the react-redux libdefs but `react-router-dom` conflicts with an existing def in webutils. We could probably resolve manually in our files.

Additionally, we'll probably need to stub out material ui, if we think it's worthwhile to do so